### PR TITLE
[SOAR-836][SOAR-1275] Pan alto remove address object from group

### PR DESCRIPTION
**Pull request by jmcadams-r7**
### Description

This PR adds the Remove Address Object from Group action to Alto Palo PAN OS 

It also introduces a bug fix and output variable to Check if Address Object in Group to allow it's output to better flow into the Remove Address Object from Group action

### Validation: 
```
rmt-mbp-5357:palo_alto_pan_os jmcadams$ icon-validate .
[*] Validating plugin at .

[*] Running Integration Validators...
[*] Executing validator HelpValidator
[*] Executing validator ChangelogValidator
[*] Executing validator RequiredKeysValidator
[*] Executing validator UseCaseValidator
[*] Executing validator SpecPropertiesValidator
[*] Executing validator SpecVersionValidator
[*] Executing validator FilesValidator
[*] Executing validator TagValidator
[*] Executing validator DescriptionValidator
[*] Executing validator TitleValidator
[*] Executing validator VendorValidator
[*] Executing validator DefaultValueValidator
[*] Executing validator IconValidator
[*] Executing validator RequiredValidator
[*] Executing validator VersionValidator
[*] Executing validator DockerfileParentValidator
[*] Executing validator LoggingValidator
[*] Executing validator ProfanityValidator
[*] Executing validator AcronymValidator
[*] Executing validator JSONValidator
[*] Executing validator OutputValidator
[*] Executing validator RegenerationValidator
[*] Executing validator HelpInputOutputValidator
[*] Executing validator SupportValidator
[*] Executing validator RuntimeValidator
[*] Plugin successfully validated!

----
[*] Total time elapsed: 411.123ms
```

#### Remove Address Object from Group
```
rmt-mbp-5357:palo_alto_pan_os jmcadams$ ./run.sh -R tests/remove_address_object_from_group.json -j
Running: cat tests/remove_address_object_from_group.json | docker run --rm   -i rapid7/palo_alto_pan_os:2.2.0  run | grep -- ^\{ | jq -r '.body | try(.log | split("\n") | .[]),.output'
Connect: Connecting..
Starting new HTTPS connection (1): pa-vm-9-0.vuln.lax.rapid7.com:443
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=keygen&user=admin&password=notpassword HTTP/1.1" 200 200
Key obtained
rapid7/Palo Alto PAN-OS:2.2.0. Step name: remove_address_object_from_group
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1hcXFHWTdyVlo5RHl3THhWK3ZzVjAzSG9vSWc9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3AvWVVZT2VEMGVPdHc0SWdCZ2NVSQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress-group%2Fentry%5B%40name%3D%27ICON+Block+List%27%5D HTTP/1.1" 200 696
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "POST /api/?type=config&action=edit&key=LUFRPT1hcXFHWTdyVlo5RHl3THhWK3ZzVjAzSG9vSWc9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3AvWVVZT2VEMGVPdHc0SWdCZ2NVSQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress-group%2Fentry%5B%40name%3D%27ICON+Block+List%27%5D&element=%3Centry+name%3D%22ICON+Block+List%22%3E%3Cstatic%3E%3Cmember%3E1.1.1.1-24%3C%2Fmember%3E%3Cmember%3E1.1.2.2%3C%2Fmember%3E%3Cmember%3EBad+IP+2%3C%2Fmember%3E%3Cmember%3ETest+Domain%3C%2Fmember%3E%3C%2Fstatic%3E%3C%2Fentry%3E HTTP/1.1" 200 76
Connect: Connecting..
Key obtained
rapid7/Palo Alto PAN-OS:2.2.0. Step name: remove_address_object_from_group

{
  "success": true
}
```
#### Check if IP in Group
```
rmt-mbp-5357:palo_alto_pan_os jmcadams$ ./run.sh -R tests/check_if_address_object_in_group.json -j
Running: cat tests/check_if_address_object_in_group.json | docker run --rm   -i rapid7/palo_alto_pan_os:2.2.0  run | grep -- ^\{ | jq -r '.body | try(.log | split("\n") | .[]),.output'
Connect: Connecting..
Starting new HTTPS connection (1): pa-vm-9-0.vuln.lax.rapid7.com:443
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=keygen&user=admin&password=notpassword HTTP/1.1" 200 200
Key obtained
rapid7/Palo Alto PAN-OS:2.2.0. Step name: check_if_address_object_in_group
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1KMkFCN0ZNdXlkbitGeTUwd3NWWHVMOG41TU09VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLSkNzWURhaWxtYkY1MU9sU1JVZkErWQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress-group%2Fentry%5B%40name%3D%27ICON+Block+List%27%5D HTTP/1.1" 200 696
Searching through 4 address objects.
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1KMkFCN0ZNdXlkbitGeTUwd3NWWHVMOG41TU09VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLSkNzWURhaWxtYkY1MU9sU1JVZkErWQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%271.1.1.1-24%27%5D HTTP/1.1" 200 536
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1KMkFCN0ZNdXlkbitGeTUwd3NWWHVMOG41TU09VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLSkNzWURhaWxtYkY1MU9sU1JVZkErWQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%271.1.2.2%27%5D HTTP/1.1" 200 204
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1KMkFCN0ZNdXlkbitGeTUwd3NWWHVMOG41TU09VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLSkNzWURhaWxtYkY1MU9sU1JVZkErWQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%27Bad+IP%27%5D HTTP/1.1" 200 164
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1KMkFCN0ZNdXlkbitGeTUwd3NWWHVMOG41TU09VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLSkNzWURhaWxtYkY1MU9sU1JVZkErWQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%27Bad+IP+2%27%5D HTTP/1.1" 200 166
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1KMkFCN0ZNdXlkbitGeTUwd3NWWHVMOG41TU09VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLSkNzWURhaWxtYkY1MU9sU1JVZkErWQ%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%27Test+Domain%27%5D HTTP/1.1" 200 161
Connect: Connecting..
Key obtained
rapid7/Palo Alto PAN-OS:2.2.0. Step name: check_if_address_object_in_group
Searching through 4 address objects.

{
  "found": true,
  "address_objects": [
    "1.1.1.1-24",
    "Bad IP 2"
  ]
}
```
#### Negative Test
```
rmt-mbp-5357:palo_alto_pan_os jmcadams$ ./run.sh -R tests/check_if_address_object_in_group.json -j
Running: cat tests/check_if_address_object_in_group.json | docker run --rm   -i rapid7/palo_alto_pan_os:3.0.0  run | grep -- ^\{ | jq -r '.body | try(.log | split("\n") | .[]),.output'
Connect: Connecting..
Starting new HTTPS connection (1): pa-vm-9-0.vuln.lax.rapid7.com:443
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=keygen&user=admin&password=notpassword HTTP/1.1" 200 200
Key obtained
rapid7/Palo Alto PAN-OS:3.0.0. Step name: check_if_address_object_in_group
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1kS29mZ3NlV0l3My9oMHBVN2s4TWp0ak1GMjg9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3hpczg5emZ5a21NRVk4bVc5L2N2Tw%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress-group%2Fentry%5B%40name%3D%27ICON+Block+List%27%5D HTTP/1.1" 200 613
Searching through 4 address objects.
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1kS29mZ3NlV0l3My9oMHBVN2s4TWp0ak1GMjg9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3hpczg5emZ5a21NRVk4bVc5L2N2Tw%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%271.1.1.1-24%27%5D HTTP/1.1" 200 536
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1kS29mZ3NlV0l3My9oMHBVN2s4TWp0ak1GMjg9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3hpczg5emZ5a21NRVk4bVc5L2N2Tw%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%271.1.2.2%27%5D HTTP/1.1" 200 204
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1kS29mZ3NlV0l3My9oMHBVN2s4TWp0ak1GMjg9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3hpczg5emZ5a21NRVk4bVc5L2N2Tw%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%27Bad+IP+2%27%5D HTTP/1.1" 200 166
/usr/local/lib/python3.7/site-packages/urllib3-1.25.3-py3.7.egg/urllib3/connectionpool.py:851: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
https://pa-vm-9-0.vuln.lax.rapid7.com:443 "GET /api/?type=config&action=get&key=LUFRPT1kS29mZ3NlV0l3My9oMHBVN2s4TWp0ak1GMjg9VlI1REg0YUdFOHA5aHMzTk9kT3JIeEVlYmwzMEVLRHcxT0crTDAwSHpLS3hpczg5emZ5a21NRVk4bVc5L2N2Tw%3D%3D&xpath=%2Fconfig%2Fdevices%2Fentry%5B%40name%3D%27localhost.localdomain%27%5D%2Fvsys%2Fentry%5B%40name%3D%27vsys1%27%5D%2Faddress%2Fentry%5B%40name%3D%27Test+Domain%27%5D HTTP/1.1" 200 161
Connect: Connecting..
Key obtained
rapid7/Palo Alto PAN-OS:3.0.0. Step name: check_if_address_object_in_group
Searching through 4 address objects.

{
  "found": false,
  "address_objects": []
}
```

### CODE REVIEW

File: palo_alto_pan_os/plugin.spec.yaml [hunk 1]
```diff
@@ -775,5 +775,48 @@ actions:
...
+        title: Address Object Name
+        description: The name of the address object name this address was found in
+        type: string
+        required: false
+  remove_address_object_from_group:
+    title: Remove Address Object from Group
+    description: Removes an address object from an address group
+    input:
+      address_object_name:
+        title: Address Object Name
```
Comment (jschipp-r7) [inline palo_alto_pan_os/plugin.spec.yaml:788]:
```suggestion
        title: Address Object
```

File: palo_alto_pan_os/help.md [hunk 2]
```diff
@@ -38,6 +38,53 @@ Example input:
...
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|success|boolean|True|Was operation successful|
+
+Example output:
+
+```
```
Comment (jschipp-r7) [inline palo_alto_pan_os/help.md:73]: Needs example output

File: palo_alto_pan_os/help.md [hunk 3]
```diff
@@ -38,6 +38,53 @@ Example input:
...
+|address_object_name|string|None|True|The name of the address object to remove|None|Malicious IP|
+|device_name|string|localhost.localdomain|True|Device name|None|localhost.localdomain|
+|group_name|string|None|True|Group name|None|ICON Block List|
+|virtual_system|string|vsys1|True|Virtual system name|None|vsys1|
+
+Example input:
+
+```
+{
+  "address_object_name": "Malicious IP",
```
Comment (jschipp-r7) [inline palo_alto_pan_os/help.md:58]: Update after schema changes

File: palo_alto_pan_os/help.md [hunk 4]
```diff
@@ -38,6 +38,53 @@ Example input:
...
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|success|boolean|True|Was operation successful|
+
+Example output:
+
+```
+```
+
+####
```
Comment (jschipp-r7) [inline palo_alto_pan_os/help.md:76]: Missing heading

File: palo_alto_pan_os/plugin.spec.yaml [hunk 5]
```diff
@@ -739,41 +739,92 @@ actions:
...
+    description: Removes an address object from an address group
+    input:
+      address_object:
+        title: Address Object
+        description: The name of the address object to remove
+        type: string
+        example: Malicious IP
+        required: true
+        order: 1
+      group_name:
```
Comment (jschipp-r7) [inline palo_alto_pan_os/plugin.spec.yaml:802]: Change this one to match the new convention in the other action `group` - "Group"

File: palo_alto_pan_os/plugin.spec.yaml [hunk 6]
```diff
@@ -739,41 +739,92 @@ actions:
...
+        order: 5
     output:
       found:
         title: Found
         description: Was address found in group
         type: boolean
         required: true
-
-
+      address_object_names:
```
Comment (jschipp-r7) [inline palo_alto_pan_os/plugin.spec.yaml:786]:
```suggestion
      address_objects:
```

File: palo_alto_pan_os/help.md [hunk 7]
```diff
@@ -66,11 +115,19 @@ Example input:
...
 |----|----|--------|-----------|
+|address_object_names|[]string|False|The names of the address objects that match or contain address|
 |found|boolean|True|Was address found in group|
 
 Example output:
 
 ```
+{
+  "found": true,
+  "address_object_names": [
```
Comment (jschipp-r7) [inline palo_alto_pan_os/help.md:126]: Regen after variable update

File: palo_alto_pan_os/help.md [hunk 8]
```diff
@@ -38,6 +38,53 @@ Example input:
...
+|group_name|string|None|True|Group name|None|ICON Block List|
+|virtual_system|string|vsys1|True|Virtual system name|None|vsys1|
+
+Example input:
+
+```
+{
+  "address_object": "Malicious IP",
+  "device_name": "localhost.localdomain",
+  "group_name": "ICON Block List",
```
Comment (jschipp-r7) [inline palo_alto_pan_os/help.md:60]: Regen after variable update

File: palo_alto_pan_os/help.md [hunk 9]
```diff
@@ -208,7 +208,7 @@ In this case, we will strip the /32 from the end and check the IP against the wh
 |object_description|string|None|False|A description for the address object|None|Blocked host from Insight Connect|
 |object_name|string|None|True|The name of the address object|None|Blocked host|
 |tags|string|None|False|Tags for the address object. Use commas to separate multiple tags|None|malware|
-|whitelist|[]string|None|False|This list contains a set of network objects that should not be blocked. This can include IPs, CIDR notation, or domains. It can not include an IP range (such as 10.0.0.0-10.0.0.10)|None|["198.51.100.100", "192.0.2.0/24", "example.com"]|
+|whitelist|[]string|None|False|This list contains a set of network objects that should not be blocked. This can include IPs, CIDR notation, or domains. It can not include an IP range (such as 10.0.0.0-10.0.0.10)|None|['198.51.100.100', '192.0.2.0/24', 'example.com']|
```
Comment (jschipp-r7) [inline palo_alto_pan_os/help.md:211]: can you change this back - former is valid JSON. icon-validate is wrong in this case, ther'es a ticket to fix it.

Comment (jmcadams-r7) [reply palo_alto_pan_os/help.md:211]: Yea, I changed it to pass validation. I'll let it fail.

**Pull request merged**